pkg/test/context.go

```diff
@@ -41,43 +37,57 @@ type TestCtx struct { //nolint:golint
 	skipCleanupOnError bool
 }
 
+// todo(camilamacedo86): Remove the following line just added for we are able to deprecated TestCtx
+// need to be done before: 1.0.0
+
+// Deprecated: Context exists for historical compatibility
+// Yoou should use Context instead of.
+type TestCtx = Context //nolint:golint
+
 type CleanupOptions struct {
-	TestContext   *TestCtx
+	TestContext   *Context
 	Timeout       time.Duration
 	RetryInterval time.Duration
 }
 
 type cleanupFn func() error
 
-func (f *Framework) newTestCtx(t *testing.T) *TestCtx {
-	// TestCtx is used among others for namespace names where '/' is forbidden and must be 63 characters or less
+func (f *Framework) newContext(t *testing.T) *Context {
+
+	// Context is used among others for namespace names where '/' is forbidden and must be 63 characters or less
 	id := "osdk-e2e-" + uuid.New()
 
 	var namespace string
 	if f.singleNamespaceMode {
 		namespace = f.Namespace
 	}
-	return &TestCtx{
-		id:                 id,
-		t:                  t,
-		namespace:          namespace,
-		namespacedManPath:  *f.NamespacedManPath,
-		client:             f.Client,
-		kubeclient:         f.KubeClient,
-		restMapper:         f.restMapper,
-		skipCleanupOnError: f.skipCleanupOnError,
```

Comment:
Removing `skipCleanupOnError` is a regression.

Reply:
It was added after the PR be created and lost in the merge. Good point.